pkg/component/autoscaling/vpa/general.go

```diff
@@ -205,12 +206,24 @@ func (v *vpa) reconcileGeneralMutatingWebhookConfiguration(mutatingWebhookConfig
 			"intentionally starts with 'zzz', such that it is called after all other webhooks which inject "+
 			"containers. All containers injected by webhooks that are called _after_ the vpa webhook will not be "+
 			"under control of vpa.")
+
+	if v.values.ClusterType == component.ClusterTypeSeed {
+		namespaceSelector = &metav1.LabelSelector{
+			MatchExpressions: []metav1.LabelSelectorRequirement{
+				{Key: "name", Operator: metav1.LabelSelectorOpNotIn, Values: []string{"kube-system"}},
```

Comment:
The key should be `kubernetes.io/metadata.name`, and there are constants that can be used here. 

```suggestion
				{Key: corev1.LabelMetadataName, Operator: metav1.LabelSelectorOpNotIn, Values: []string{metav1.NamespaceSystem}},
```

Reply:
Done. Thanks for pointing me to the specific constant definitions!